mixitup/teamInfo.js

```diff
@@ -0,0 +1,247 @@
+var context = 
```

Comment:
Which source of team data are we actually using here? And I thought we were going to try and get it to draw on the Google Spreadsheet directly via the JSON view?